packages/blocks/src/__internal__/clipboard/utils/pure.ts

```diff
@@ -102,6 +106,25 @@ export function performNativeCopy(items: ClipboardItem[]): boolean {
   return success;
 }
 
+function _copyImage(imageSrc: string): boolean {
```

Comment:
We don't use `_foo` for module-level functions, only `this._foo` for private class methods and properties.